analytics/find-hds-react-version-usage.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# Call to Github API to find all occurrences of "hds-react" under City-of-Helsinki org.
+curl -H "Accept: application/vnd.github+json"  -H "Authorization: token $GITHUB_TOKEN" "https://api.github.com/search/code?q=hds-react+in:file+filename:package.json+org:City-of-Helsinki" | jq '[.items[] | {name: .repository.name, html_url: .repository.html_url, package_url: .git_url}]' > hds-react-version-usage.json
```

Comment:
Small improvement idea: Since the GITHUB_TOKEN is used in this line and also at line 13, we could maybe store it as a global variable in the beginning of the file if possible? Then (user who has not setup the env variable) user would only have to replace the variable to one place